isotovideo

```diff
@@ -322,18 +322,27 @@ if ($ENV{RUN_DEBUGVIEWER}) {
 
 use IO::Select;
 
-my $s = IO::Select->new();
-$s->add($testfd);
-$s->add($cfd);
-$s->add($bmwqemu::backend->{from_child});
+my $io_select = IO::Select->new();
+$io_select->add($testfd);
+$io_select->add($cmd_srv_fd);
+$io_select->add($bmwqemu::backend->{from_child});
 
 # now we have everything, give the tests a go
 $testfd->write("GO\n");
 
-my $interactive = 0;
-my $needinput   = 0;
-
+# store the name of the current test
 my $current_test_name;
+my $current_test_full_name;
+
+# store the name of the test where the test execution should be paused
+my $pause_test_name = $bmwqemu::vars{PAUSE_AT};
+
+# when paused, stop passing commands to the backend
```

Comment:
I wonder if this should be a property of the test variables? so that the worker/webui comunication can easily tell if isotovideo is accepting or not more commands (on top of the job state...)

Reply:
Ok, maybe later. Not sure how I can add/update test variables from `isotovideo` so the changes are promoted to the child processes and the web UI.